### PR TITLE
New package: SDL_gfx-2.0.25

### PR DESCRIPTION
**Pull request by pullmoll**
Required for widelands, which is next on my list.

### CODE REVIEW

File: srcpkgs/SDL_gfx/template [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+short_desc="Graphics drawing primitves for SDL"
+maintainer="Jürgen Buchmüller <pullmoll@t-online.de>"
+license="BSD"
+homepage="http://cms.ferzkopp.net/index.php/software/13-sdl-gfx"
+distfiles="http://www.ferzkopp.net/Software/SDL_gfx-2.0/SDL_gfx-2.0.25.tar.gz"
+checksum=556eedc06b6cf29eb495b6d27f2dcc51bf909ad82389ba2fa7bdc4dec89059c0
+
+pre_configure() {
+	case "${XBPS_TARGET_MACHINE}" in
+		x86_64*)
```
Comment (xtraeme) [inline srcpkgs/SDL_gfx/template]: `x86_64*|i686*) --disable-mmx` ...

File: srcpkgs/SDL_gfx/template [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+			configure_args="${configure_args} --disable-mmx"
+			;;
+	esac
+}
+post_install() {
+	vlicense COPYING
+	vlicense LICENSE
+}
+SDL_gfx-devel_package() {
+	depends="SDL-devel ${sourcepkg}>=${version}"
```
Comment (xtraeme) [inline srcpkgs/SDL_gfx/template]: `${sourcepkg}-${version}_${revision}`

Comment (pullmoll) [inline srcpkgs/SDL_gfx/template]: I took line from the SDL_image template, so this and probably others could need a fix as well?

Comment (xtraeme) [inline srcpkgs/SDL_gfx/template]: Yes, but those pkgs were modified long ago... not all them were updated :-)

File: srcpkgs/SDL_gfx/template [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+maintainer="Jürgen Buchmüller <pullmoll@t-online.de>"
+license="BSD"
+homepage="http://cms.ferzkopp.net/index.php/software/13-sdl-gfx"
+distfiles="http://www.ferzkopp.net/Software/SDL_gfx-2.0/SDL_gfx-2.0.25.tar.gz"
+checksum=556eedc06b6cf29eb495b6d27f2dcc51bf909ad82389ba2fa7bdc4dec89059c0
+
+pre_configure() {
+	case "${XBPS_TARGET_MACHINE}" in
+		x86_64*|i686*)
+			configure_args="${configure_args} --enable-mmx"
```
Comment (xtraeme) [inline srcpkgs/SDL_gfx/template]: `configure_args+=" --enable-mmx"`

File: srcpkgs/jbigkit/template [hunk 4]
```diff
@@ -11,9 +11,8 @@ distfiles="http://www.cl.cam.ac.uk/~mgk25/download/jbigkit-$version.tar.gz"
 checksum=de7106b6bfaf495d6865c7dd7ac6ca1381bd12e0d81405ea81e7f2167263d932
 
 do_build() {
-	CFLAGS+=-I${wrksrc}/libjbig make -e
+	make CC=$CC CFLAGS="${CFLAGS} -I${wrksrc}/libjbig ${LDFLAGS}" ${makejobs}
```
Comment (xtraeme) [inline srcpkgs/jbigkit/template]: This change is unrelated to this PR.

Comment (pullmoll) [inline srcpkgs/jbigkit/template]: github didn't want to let me git push and git told me to git pull first. Not sure what to do to get rid of the unwanted part.

File: srcpkgs/SDL_gfx/template (hunk 3 above)
Comment (pullmoll) [inline srcpkgs/SDL_gfx/template]: Oh, didn't know bashisms were ok.

File: srcpkgs/jbigkit/template (hunk 4 above)
Comment (pullmoll) [inline srcpkgs/jbigkit/template]: Well, wtf, I'll re-fork and create a clean new pull request :)
